crates/rethnet_eth/src/remote/cacheable_method_invocation.rs

```diff
@@ -0,0 +1,154 @@
+use crate::remote::methods::{GetLogsInput, MethodInvocation, Percentiles, TransactionInput};
```

Comment:
In a next PR, I'd recommend moving the RPC client and caching related files out into a `rethnet_rpc_client` crate

Reply:
Ok, added follow up issue: https://github.com/NomicFoundation/rethnet/issues/161